ring/replication_set_test.go

```diff
@@ -241,3 +241,60 @@ func TestReplicationSet_Do(t *testing.T) {
 		})
 	}
 }
+
+func TestHasReplicationSetChangedWithoutStateIgnoresTimeStampAndState(t *testing.T) {
+	// Only testing difference to underlying Equal function
+	rs1 := ReplicationSet{
```

Comment:
[nit] May calling this `initialState` (or something like this) may clarify this is the initial state we're comparing to?

Reply:
fair